props/saber_caiwyn_buttons.h

```diff
@@ -0,0 +1,604 @@
+// Blaster Block: Click Aux
+// Lightning Block: Hold Aux
+// Lockup: Hold Aux during impact
+// Drag: Hold Aux during impact with saber pointed down.
+// Melt: Hold Aux and stab
+// Enter Volume Menu: Double-click and hold Aux (Be aware that the first click will trigger a blaster block)
+//   Increase Volume: Rotate hilt right while in Volume Menu
+//   Decrease Volume: Rotate hilt left while in Volume Menu
+//   Save Changes and Exit Volume Menu: Click Power
+//   Discard Changes and Exit Volume Menu: Click Aux
+// Enter Color Change Mode (Color Wheel): Triple-click and hold Aux (Be aware that the first two clicks will trigger blaster blocks)
+//   Change Color: Rotate hilt while in Color Change Mode
+//   Color Zoom: Press & Hold Power and Rotate hilt while in Color Change Mode
+//   Save Changes and Exit Color Change Mode: Release Power
+//   Reset Color and Exit Color Change mode: Click Aux
+// Deactivate Saber: Hold Aux or Aux2 and press Power
+//
+// You will need the following sound files in order for menus to work properly:
+// vmbegin.wav - Enter Volume Change Menu
+// vmend01.wav - Save Volume Change
+// vmend02.wav - Cancel Volume Change
+// monce.wav - Set Track Player to play a single track one time
+// mloop.wav - Set Track Player to repeat a single track
+// mrotate.wav - Set Track Player to repeat all tracks
+// ccbegin.wav - Enter Color Change Mode
+// ccend01.wav - Save Color and Exit Color Change Mode
+// ccend02.wav - Reset Color and Exit Color Change Mode
+//
+// Options you can add to your config file:
+// #define DISABLE_BEEPS        - Disables the single beep that occurs when selecting fonts, tracks, or entering or exiting the volume and color change menus.
+//                                Note that beeps will still play if any sound file listed above is missing.
+// #define DISABLE_COLOR_CHANGE - Disables the color change menu.
+// #define CUSTOM_BATTERY_LEVEL - To use your own sound files when checking battery level instead of Proffie's built-in voice.
+//                                This will require additional sound files: mbatt.wav, mpercent.wav, mzero.wav, mnum1.wav - mnum20.wav, thirty.wav - ninety.wav, hundred.wav
+// #define DISABLE_BUTTON_CLASH - When you click the power button, both a clash sound and a lockup are simultaneously triggered.
+//                                This is because I have found that quickly starting and ending a lockup often sounds like multiple clashes rather than a single clash.
+//                                To smooth everything out, a clash sound is mixed in as well.  This works for most fonts, but if it sounds weird you can disable the clash sound and just use the lockup by itself.
+
+#ifndef PROPS_CAIWYN_BUTTONS_H
+#define PROPS_CAIWYN_BUTTONS_H
+
+#include "prop_base.h"
+#include "../sound/hybrid_font.h"
+#include "../sound/sound_library.h"
+
+#undef PROP_TYPE
+#define PROP_TYPE CaiwynButtons
+
+#ifndef NUM_BUTTONS
+#define NUM_BUTTONS 2
```

Comment:
This is a weird thing to do in a prop file, I suggest an #error instead.

Reply:
I could've sworn I copied that from an existing prop file but I'm not seeing it anywhere else now.  Probably should just remove it.

Reply:
Done, this is removed.